spec/lib/rmt/scc_spec.rb

```diff
@@ -253,7 +251,7 @@ def scc
           FakeFS.with_fresh do
             FileUtils.mkdir_p path
             described_class.new.export(path)
-            expect(File.exist?(File.join(path, "organizations_#{data}.json")))
+            expect(File).to be_exist(File.join(path, "organizations_#{data}.json"))
```

Comment:
Apparently that `expect` didn't do anything, Rubocop raised:
```
RSpec/VoidExpect: Do not use expect() without .to or .not_to. Chain the methods or remove it. (http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/VoidExpect)
```

Reply:
`expect(File).to exist(File.join(path, "organizations_#{data}.json"))`

No need for `be_`, since the [exists](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/exist-matcher) matcher... exists.

Reply:
Rubocop auto-fixed it from `File.exist?` to `be_exist` :D